MongoDB Atlas Solution

stevelordbq opened the pull request:
Required items, please complete
   
   Change(s):
   - Initial version of Solution contain MongoDB Atlas Administration API connector for Logs

   Reason for Change(s):
   - Initial version

   Version Updated:
   - Required only for Detections/Analytic Rule templates

   Testing Completed:
   - No

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes - one false positive

stevelordbq commented:
> @stevelordbq please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree company="MongoDB"

v-maheshbh commented:
Hi @stevelordbq JSON does not support comments, Kindly remove the comment line from azure deploy file.
<img width="964" height="171" alt="image" src="https://github.com/user-attachments/assets/6c0a1eaa-eb10-4db5-babd-69c9e2d46707" /> and To fix the logo validation issue, open the logo file in Visual Studio Code and look for the id property. Replace it with a new GUID, which is a 16-character unique identifier. You can generate a GUID online and update the id property with it.
Thanks!

stevelordbq commented:
> Hi @stevelordbq JSON does not support comments, Kindly remove the comment line from azure deploy file.  and To fix the logo validation issue, open the logo file in Visual Studio Code and look for the id property. Replace it with a new GUID, which is a 16-character unique identifier. You can generate a GUID online and update the id property with it. Thanks!
I have removed the comments @v-maheshbh .
I'm a bit confused by the svg id comment. The validation error is stating the file contains an xmlns:xlink link, which it does. I have now removed that. There are two id fields in the file <g id> and <clipPath id>. I'm unsure which, if either, I should set to a GUID value. I'll resubmit with just the xmlns:xlink link removed and see if it clear the error.

stevelordbq commented:
Many thanks @v-maheshbh. I thought I had created file MDBALogTable_CL with extension json, but obviously not. Well spotted.

stevelordbq commented:
CLA is agreed but check shows it isn't. Is there something I am missing @v-maheshbh ?

stevelordbq commented:
Hi @v-maheshbh . I see you have updated my PR. Thank you. Aside from the Solution_MongoDBAtlas.json escape character change, is there anything else you spotted?

QQ, if I update Solution_MongoDBAtlas.json etc should I regenerate Package/ again and submit the new version as part of the PR, or should I be generating Package/ at all? It looks there are several checks that are skipping files as it doesn't think they are from the same origin.

v-maheshbh commented:
Hi @stevelordbq I don’t have access to the Invocation logs link. Could you please share a screenshot of the logs instead?
Thnaks!

<img width="596" height="188" alt="image" src="https://github.com/user-attachments/assets/00b681dc-bcb4-40f1-80e8-353847404705" />

v-maheshbh commented:
Hi @stevelordbq Thank you for providing the logs. Kindly proceed to accept the Contributor License Agreement (CLA) by commenting:
@microsoft-github-policy-service agree company="MongoDB"

Thanks!

v-maheshbh commented:
Hi @DianaMDB Yes, please proceed with accepting the CLA. A new pull request can be submitted with the changes at a later time.
Thanks!

DianaMDB commented:
@microsoft-github-policy-service agree company="MongoDB"
